### ISSUE TITLE
Design Intent - Accessibility Feedback - mhv-secure-messaging, OH Curated List

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#103504]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry again for the delayed written feedback. Please let me know if you have any questions or concerns, or if my delay impacted your team's work.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** As you finalize your content, on the [Message details page](https://www.figma.com/design/uddjxccqvUyz5FSg1aD5wi/Z---Triage-Group-Flows?node-id=2902-20885&t=PNxjV67nS59e06it-4) I think the "Edit" link to change health care systems is a little ambiguous. Something more descriptive like "Change VA health care systems" will probably work better here.
- [ ] **Consider:** Cognitive load in the combobox is a potential concern. I'll suggest two possible routes to explore (together or on their own):
   - Consider using [`optgroup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) in the dropdown to group related options together. It should make it more scannable, and assistive technology support for `optgroup` is fine (could be a little stronger but is fine to use).
   - If the combobox is going to have an absurdly large number of items in it, consider adding additional filtering questions besides just choosing a health care system. I realize this probably changes the scope of the work a bit, but if nothing else I think it's worth exploring as you do usability testing --- asking users if they would want additional filtering questions, and/or if they have criteria already in mind.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
